app/views/pg_hero/home/_pev2.html.erb

```diff
@@ -0,0 +1,27 @@
+<script src="https://unpkg.com/vue@3.2.45/dist/vue.global.prod.js"></script>
+<script src="https://unpkg.com/pev2/dist/pev2.umd.js"></script>
+<link
+  href="https://unpkg.com/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+  rel="stylesheet"
+/>
+<link rel="stylesheet" href="https://unpkg.com/pev2/dist/style.css" />
+
+<div id="pev2" class="d-flex flex-column" style="height: 50vh">
```

Comment:
Styles look good to me! Just wondering what `d-flex` is though? I can't tell if it's a typo of a special type of flexbox 😅

Reply:
> Just wondering what `d-flex` is though?

Maybe a bootstrap thing? https://getbootstrap.com/docs/4.0/utilities/flex/

Reply:
Ohh true, I forgot the admin uses bootstrap

Reply:
Yeh, but also this thing includes it's own bootstrap on line 7 :sweat_smile:

I literally pulled this code verbatim from https://github.com/dalibo/pev2?tab=readme-ov-file#without-building-tools as a hacky way to see if it worked.

It feels very ugly, I think pghero is already in an iframe in the admin, and then this partial is also loading bootstrap and vuejs :vomiting_face: 

I don't have the FE skills  to do this cleaner, so open to suggestions.  Maybe it doesn't matter if it's ugly :shrug:

Reply:
Actually, I don't think our admin uses bootstrap haha. So at least it's not double loaded.
I feel like this is fine, if there is no other simple way to imbed pev2 without vue then this is probably the best way to do it without spending ages on it.